src/main/java/de/retest/recheck/RecheckOptions.java

```diff
@@ -256,17 +257,17 @@ RecheckOptionsBuilder setIgnore( final Filter filter ) {
 		 * Appends a filter to the default filters. Cannot be used once a filter is overwritten with
 		 * {@link #setIgnore(String)}.
 		 *
-		 * @param filtername
-		 *            The filter to add to the ignore.
+		 * @param filterName
+		 *            The name of the filter (including .filter extension) to use for printing the differences.
 		 * @return self
 		 * @see #setIgnore(String)
 		 */
-		public RecheckOptionsBuilder addIgnore( final String filtername ) {
+		public RecheckOptionsBuilder addIgnore( final String filterName ) {
 			if ( ignoreFilter == null ) {
-				ignoreFilterToAdd.add( getFilterByName( filtername ) );
+				ignoreFilterToAdd.add( getFilterByName( filterName ) );
 			} else {
 				throw new IllegalStateException(
-						"Cannot combine `setIgnore(filtername)` and `addIgnore(filtername)`." );
+						"Cannot combine `setIgnore(filterName)` and `addIgnore(filterName)`." );
```

Comment:
Methods are uniquely defined based on a) their name and b) parameter types (not names):

```suggestion
						"Cannot combine `setIgnore( String )` and `addIgnore( String )`." );
```